tools/reviewer/webapp/README.md

```diff
@@ -1,23 +1,32 @@
-## You need installed
-Required to use Reviewer locally:
-[node](https://nodejs.org/) LTS version
-[npm](https://www.npmjs.com/)  
-[protoc](https://github.com/protocolbuffers/protobuf/releases)  
-Optional:  
-[firebase](https://firebase.google.com/docs/hosting/quickstart) to be able to deploy.  
-[angular](https://angular.io/) to use `ng` features.  
-
-## For the first time
+# Reviewer webapp
+The Reviewer front-end is an Angular webapp.
+
+To set up your development environment, follow these steps:
+
+## Installation
+Install these:
+* [node](https://nodejs.org/) LTS version
+* [npm](https://www.npmjs.com/)  
+* [protoc](https://github.com/protocolbuffers/protobuf/releases)  
+* `google-protobuf 3.5` or above.
```

Comment:
protoc and google-protobuf are the same app. Maybe we could put them on one line? (with one *)

Reply:
Done